README.md

```diff
@@ -33,4 +33,4 @@ If you are interested in contributing to the website, please check out our Contr
 
 ## Code of Conduct
 
-As a contributor please follow the Code of Conduct to keep the community open and inclusive. Please read and follow the [PyCon India Code of Conduct](https://in.pycon.org/2020/coc/) which governs the overall conduct for the conference.
\ No newline at end of file
+As a contributor please follow the Code of Conduct to keep the community open and inclusive. Please read and follow the [PyCon India Code of Conduct](https://in.pycon.org/2020/coc/) which governs the overall conduct for the conference.
```

Comment:
The link should be https://in.pycon.org/2021/coc/

Reply:
You're right. Done.